analyzer/src/funTest/kotlin/GradleTest.kt

```diff
@@ -132,6 +136,7 @@ class GradleTest : StringSpec() {
             yamlMapper.writeValueAsString(result) shouldBe expectedResult
         }
 
+        // Disabled as it causes hangs and memory issues.
```

Comment:
Maybe add "... on CI", IIRC we have only seen it there, never locally.

Reply:
Done.